src/admin-views/admin-update-message.php

```diff
@@ -30,11 +30,11 @@
 
 	<div class="tribe-half-column">
 		<h2><?php printf( esc_html__( 'Keep the Core Plugin %sFREE%s!', 'the-events-calendar' ), '<strong>', '</strong>' ); ?></h2>
-		<p><?php printf( esc_html__( 'Every time you rate %s5 stars%s, a fairy is born. Okay maybe not, but more happy users mean more contributions and help on the forums. The community NEEDS your voice.', 'the-events-calendar' ), '<strong>', '</strong>' ); ?></p>
+		<p><?php printf( esc_html__( 'Every time you rate %s5 stars%s, a fairy is born. Okay, maybe not, but more happy users means more contributions and help on the forums. The community NEEDS your voice.', 'the-events-calendar' ), '<strong>', '</strong>' ); ?></p>
 		<p><a href="https://wordpress.org/support/plugin/the-events-calendar/reviews/?filter=5" target="_blank" class="button-primary"><?php esc_html_e( 'Rate It', 'the-events-calendar' ); ?></a></p>
 
 		<br/>
-		<h2><?php esc_html_e( 'PSST... Want a Discount?', 'the-events-calendar' ); ?></h2>
+		<h2><?php esc_html_e( 'PSST...Want a Discount?', 'the-events-calendar' ); ?></h2>
```

Comment:
can we get a language and style guide ruling around ellipses to indicate a pause? This is not my preferred spacing, and I'm not seeing an "official" stance anywhere for informal usage.

Reply:
`&hellip;` is an option instead of `...`
https://www.toptal.com/designers/htmlarrows/punctuation/horizontal-ellipsis/

But George can look up Chicago styling's suggestion. I'd guess a space after is preferable.

Reply:
Good call. I've adjusted the Tribe Style Guide to include references to our style manual of choice (Chicago), which point to having **no spaces** in this kind of case.

Screenshots of the references etc. can be found here in the Tribe Style Guide now: http://inside.tri.be/tribe_kb/the-modern-tribe-style-guide/#ellipses